docs/configfile.adoc

```diff
@@ -0,0 +1,52 @@
+== Configuration file
+
+You can use smart-testing.y(a)ml file which is present at `${project.dir}/smart-testing.yml`.
+Configuration file looks as follows:
+
+[[config-file]]
+[source, yaml, indent=0]
+----
+include::../core/src/test/resources/configuration/smart-testing.yml[]
+----
+copyToClipboard:config-file[]
+
+However you can overwrite all configuration options using system properties supported by Smart Testing. You can look <<_reference_card, references>> for all supported system properties.
+
+=== Configuration file reference
+The `const:core/src/main/java/org/arquillian/smart/testing/configuration/Configuration.java[name="SMART_TESTING_YML"]` file is a `YAML` file defining required configuration to configure Smart Testing.
+
+NOTE: You can use either a .yml or .yaml extension for this file.
+
+==== strategies
```

Comment:
instead of creating a new titli for every param I would use the same approach of explaining parameters as in cube config: http://arquillian.org/arquillian-cube/#_configuration

Reply:
you know, referencing from the config file example using numbers